### DISCUSSION TITLE
Pitch Bend command

### DISCUSSION THREAD

**Discussion by Bipman1** [mido/mido#467] [Q&A] [answered]
Hi

I'm trying to user the pitch bend command but can't work out the actual command.  I've tried 'Pitch_Bend' and 'PitchBend' without success.  What should I be using?   Also, is there a list of commands that are different from the MIDI spec ones?

Bipman

**Comment by rdoursenaud** [comment, marked as answer]
The name is `pitch_wheel` in mido. You'll find all the supported messages in the documentation: https://mido.readthedocs.io/en/latest/message_types.html?highlight=pitch#supported-messages

**Comment by Bipman1** [comment]
Thanks !

On Mon, 13 Mar 2023 at 07:59, Raphaël Doursenaud ***@***.***>
wrote:

> The name is pitch_wheel in mido. You'll find all the supported messages
> in the documentation:
> https://mido.readthedocs.io/en/latest/message_types.html?highlight=pitch#supported-messages
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/mido/discussions/467#discussioncomment-5288179>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AY2WDJGC5NUNC3ZALSATESLW33HVNANCNFSM6AAAAAAVXRBB2Y>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 

Regards


Stephen Gray



Check out the links below for Skynet ambient music

YouTube
<https://www.youtube.com/channel/UC4cOaAR33QRWzmYxr44Uu-Q?view_as=subscriber>
https://SoundCloud.com\Stephen-Gray-260393121
